packages/playground/src/weblets/tf_nextcloud.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <weblet-layout
+    ref="layout"
+    @mount="layoutMount"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
+    :disk="disks.reduce((total, disk) => total + disk.size, rootFilesystemSize)"
+    :ipv4="ipv4"
+    :certified="certified"
+    :dedicated="dedicated"
+    title-image="images/icons/nextcloud.png"
```

Comment:
```suggestion
  <weblet-layout
    ref="layout"
    :cpu="solution?.cpu"
    :memory="solution?.memory"
    :disk="disks.reduce((total, disk) => total + disk.size, rootFilesystemSize + (solution?.disk ?? 0))"
    :ipv4="ipv4"
    :certified="certified"
    :dedicated="dedicated"
    title-image="images/icons/nextcloud.png"
```

1. We don't need to use `layoutMount` here as will be shown later
2. We need to add disk size in count in order to update price calculator based on the new value `rootFilesystemSize + (solution?.disk ?? 0)`